scripts/scil_tractogram_alter.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+"""
+This script is used to alter a bundle to reach specific minimum dice
+coefficient from the original bundle. The script will subsample, trim, cut,
+replace or transform the streamlines until the minimum dice is reached.
+(cannot be combined in one run, use the script multiple times if needed)
```

Comment:
This is not clear for me. What cannot be combined? Do you mean Cannot be used on multiple bundles in one run? Or the various operations choices cannot be combined?

Reply:
various operations choices cannot be combined?